docs/cli.txt

```diff
@@ -67,3 +67,41 @@ You can get a list of commands by typing the command ``\?``:
 | ``\q``               | Quit Crate shell                                                                   |
 +----------------------+------------------------------------------------------------------------------------+
 
+
+===============
+Status Messages
+===============
+
+The ``SUCCESS`` status messages are only printed when ``crash`` is used as
+an interactive tty.
+
+If the output is routed, e.g. dumped into a file or piped into another command
+(such as `jq`_) for further processing of the response, these messages are omitted.
+
+
+The synopsis of the ``SUCCESS`` message is as follows::
+
+    SUCCESS <COMMAND>, <NUM> row(s) affected (<DURATION> sec)
+
+If the SQL statement (``SELECT``) returns selected columns::
+
+    SUCCESS <COMMAND> <NUM> row(s) in set (<DURATION> sec)
+
+The ``COMMAND`` is the definition/manipulation/query keyword, such as
```

Comment:
not sure if we should call it command here, in crate we usually call it statement I guess.